core/src/test/java/com/linecorp/armeria/server/logging/LoggingServiceTest.java

```diff
@@ -154,6 +155,165 @@ public void infoLevel() throws Exception {
                             ", trailers: " + RESPONSE_TRAILERS);
     }
 
+    @Test
+    public void  map_request_level_test_header_as_warn() throws Exception {
+        when(log.requestHeaders()).thenAnswer(invocation -> RequestHeaders.of(HttpMethod.GET, "/",
+                                                                              "x-req", "test",
+                                                                              "x-res", "test"));
+
+        final LoggingService service =
+                LoggingService.builder()
+                              .requestLogLevelMapper(log -> {
+                                  if (log.requestHeaders().contains("x-req")) {
+                                      return LogLevel.WARN;
+                                  } else {
+                                      return LogLevel.INFO;
+                                  }
+                              })
+                              .responseLogLevelMapper(log -> {
+                                  if (log.requestHeaders().contains("x-res")) {
+                                      return LogLevel.WARN;
+                                  } else {
+                                      return LogLevel.INFO;
+                                  }
+                              })
+                              .newDecorator().apply(delegate);
+
+        service.serve(ctx, REQUEST);
+        verify(logger).warn(REQUEST_FORMAT,
+                            "headers: " + REQUEST_HEADERS + ", content: " + REQUEST_CONTENT +
+                            ", trailers: " + REQUEST_TRAILERS);
+        verify(logger).warn(RESPONSE_FORMAT,
+                            "headers: " + RESPONSE_HEADERS + ", content: " + RESPONSE_CONTENT +
+                            ", trailers: " + RESPONSE_TRAILERS);
+    }
+
+    @Test
+    public void map_request_level_header_as_info() throws Exception {
```

Comment:
- Ditto - camel case
- Could this method be merged with `map_request_level_test_header_as_warn` or could we deduplicate the log mappers?